docs/cockpit.rst

```diff
@@ -23,19 +23,19 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Start cockpit
+2. Start Cockpit
 
-   ``$ vagrant service-manager restart cockpit``
+   ``$ vagrant service-manager start cockpit``
```

Comment:
I am not opposed to a `service-manager start/stop`, but still don't like that we use it to control cockpit. For now, I think we would be better of to use `systemctl` for that.